### DISCUSSION TITLE
add strip parmater to String

### DISCUSSION THREAD

**Discussion by s3rgeym** [collerek/ormar#653] [Ideas]
```python
first_name: str = ormar.String(min_length=3, max_length=31, strip=True)
...
value = value.strip()
```

**Comment by SepehrBazyar** [comment]
Hi @tz4678 
I think that to apply such changes, adding a parameter to the field is not an interesting method.

Since **ormar** uses **pydantic** for data validation, the examples in [this link](https://pydantic-docs.helpmanual.io/usage/validators/) will be form pydantic's own documentation.
